src/components/common/animations/ClaimButton/ClaimButton.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import Lottie from 'lottie-react-native'
+import { Platform, TouchableOpacity } from 'react-native'
+import _set from 'lodash/set'
+import animationData from './data.json'
+
+class ClaimButton extends React.Component {
```

Comment:
why you've introduced new components into the app as the classes ?
React team is planning to stop supporting classes next releases.

So please, don't use classes anymore. Use functional components & hooks please

Reply:
@serdiukov-o-nordwhale we decided to use Clases witing lottie library as the function component have some issue wit lottie animations.